Description: Fix crash on disconnect for TigerVNC 1.9
Author: Joachim Falk <joachim.falk@gmx.de>

See upstream bug #812 (https://github.com/TigerVNC/tigervnc/issues/812)
and https://github.com/TigerVNC/tigervnc/commit/3fed95eda27dfbeee6535f987f5d14a66f64749b#diff-cad379b518c09467c09520f2f863c6d1R165
for further discussion. Moreover, this should already be fixed in master.

Index: pkg-tigervnc/unix/xserver/hw/vnc/XserverDesktop.cc
===================================================================
--- pkg-tigervnc.orig/unix/xserver/hw/vnc/XserverDesktop.cc
+++ pkg-tigervnc/unix/xserver/hw/vnc/XserverDesktop.cc
@@ -148,7 +148,7 @@ XserverDesktop::~XserverDesktop()
     listeners.pop_back();
   }
   while (!httpListeners.empty()) {
-    vncRemoveNotifyFd(listeners.back()->getFd());
+    vncRemoveNotifyFd(httpListeners.back()->getFd());
     delete httpListeners.back();
     httpListeners.pop_back();
   }
